README.md

```diff
@@ -87,10 +87,20 @@ npm start
   ]
 ]
 ```
+
+* `CONNECTOR_NOTIFICATION_VERIFY` (default: `'true'`) The connector verifies the signatures on notifications.
```

Comment:
If Connector's running in production, I wonder if this should be configurable.
It seems to me that, in production, this should be always enforced - Connector must always verify notifications.